draft-ietf-quic-transport.md

```diff
@@ -2712,8 +2712,11 @@ An implementation of QUIC might provide applications with an option to defer an
 idle timeout.  This facility could be used when the application wishes to avoid
 losing state that has been associated with an open connection, but does not
 expect to exchange application data for some time.  With this option, an
-endpoint could send a PING frame periodically to defer an idle timeout; see
-{{frame-ping}}.
+endpoint could send a PING frame ({{frame-ping}}) periodically, which will cause
+the peer to restart its idle timeout period.  Sending a packet containing a PING
+frame also restarts the idle timeout for the endpoint if this is the first
+ack-eliciting packet sent since receiving a packet or when an acknowledgment is
```

Comment:
I'm confused about this last part.  How you receive an acknowledgement without receiving a packet?

Reply:
So it says that the idle time is restarted when an acknowledgment is received.  Of course you need a packet for that, which is why the idle timeout restarts.